ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobPropertiesSerDe.java

```diff
@@ -83,5 +86,7 @@ public static void serializeBlobProperties(ByteBuffer outputBuffer, BlobProperti
     Utils.serializeNullableString(outputBuffer, properties.getContentType());
     Utils.serializeNullableString(outputBuffer, properties.getOwnerId());
     Utils.serializeNullableString(outputBuffer, properties.getServiceId());
+    outputBuffer.putShort(properties.getAccountId());
```

Comment:
Maybe I had a misunderstanding, but seems `aid` and `cid` are serialized last, but deserialized first in case `Version2`? Would this be a problem?